dm/pkg/gtid/gtid.go

```diff
@@ -80,14 +84,37 @@ func ParserGTID(flavor, gtidStr string) (Set, error) {
 	case mysql.MariaDBFlavor:
 		m = &MariadbGTIDSet{}
 	case mysql.MySQLFlavor:
+		// check for xxx:0
+		if IsNilMySQLGTIDSet(gtidStr) {
+			log.L().Warn("get empty gtid set end with `0`", zap.String("gtid", gtidStr))
+			return MinGTIDSet(mysql.MySQLFlavor), nil
+		}
 		m = &MySQLGTIDSet{}
 	default:
 		return nil, terror.ErrNotSupportedFlavor.Generate(flavor)
 	}
+
 	err = m.Set(gtid)
 	return m, err
 }
 
+// check whether a gtid set is nil(start sync from start)
+// mysql: uuid:0
+// mariadb: x-y-0(no need to handle)
```

Comment:
did you test that mariaDB supports x-y-0?

Reply:
test with 0-0-0, update comment in 044a3e3